src/taipy/gui/gui.pyi

```diff
@@ -227,6 +226,8 @@ class Gui:
     def _register_data_accessor(self, data_accessor_class: t.Type[_DataAccessor]) -> None: ...
     def get_flask_app(self) -> Flask: ...
     def _set_frame(self, frame: FrameType): ...
+    __css_file: Incomplete
```

Comment:
???

Reply:
I've seen that while backporting ...
I'll move it back to where it was ...